app/assets/javascripts/discourse/app/components/composer-editor-uppy.js

```diff
@@ -1,6 +1,12 @@
 import ComposerEditor from "discourse/components/composer-editor";
+import { alias } from "@ember/object/computed";
 import ComposerUploadUppy from "discourse/mixins/composer-upload-uppy";
 
 export default ComposerEditor.extend(ComposerUploadUppy, {
   layoutName: "components/composer-editor",
+  fileUploadElementId: "file-uploader",
+  eventPrefix: "composer",
+  uploadType: "composer",
+  uppyId: "composer-editor-uppy",
+  composerModel: alias("composer"),
```

Comment:
Should that top level component unsubscribe from `this.messageBus.unsubscribe("/uploads/composer");` when the component is destroyed?
  
I don't see in code moment when we start listening to that queue, so probably it was some unused code but I would like to confirm with you

Reply:
Yeah I removed that messageBus unsubscribe because I couldn't see the channel being used anywhere, there is nothing subscribing to it.